src/Core/Tools/SendFeatures/Services/SendFileSettingHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Bit.Core.Tools.Services;
+
+namespace Bit.Core.Tools.SendFeatures;
+
+public class SendFileSettingHelper : ISendFileSettingHelper
+{
+    private readonly long _fileSizeLeeway = 1024L * 1024L; // 1MB
+    private readonly long MAX_FILE_SIZE = Constants.FileSize501mb;
+    private readonly string MAX_FILE_SIZE_READABLE = "500 MB";
```

Comment:
⛏️ Seems like this constant should be referred to in the `SendsController` when the file size is too big.

Reply:
Made this happen with changing SendFileSettingHelper to a static class.